ecosystem/sep-0010.md

```diff
@@ -195,7 +199,10 @@ The verification process confirms that the **Client** controls the **Client Acco
 Upon successful verification, **Server** responds with a session JWT, containing the following claims:
 
 * `iss` (the principal that issued a token, [RFC7519, Section 4.1.1](https://tools.ietf.org/html/rfc7519#section-4.1.1)) — a [Uniform Resource Identifier (URI)] for the issuer (`https://example.com` or `https://example.com/G...`)
-* `sub` (the principal that is the subject of the JWT, [RFC7519, Section 4.1.2](https://tools.ietf.org/html/rfc7519#section-4.1.2)) — the public key of the authenticating Stellar account (`G...`)
+* `sub` (the principal that is the subject of the JWT, [RFC7519, Section 4.1.2](https://tools.ietf.org/html/rfc7519#section-4.1.2)) — there are three possible formats:
+  * If the **Client Account** is a muxed account, the `sub` value should also be the muxed account (`M...`).
+  * If a memo was attached to the challenge transaction, the `sub` should be the Stellar account appended with the memo, separated by a colon (`G...:17509749319012223907`).
+  * If neither of the above conditions are true, the `sub` value should be Stellar account (`G...`).
```

Comment:
💡 Some grouping of the formats might make this easier to pass and digest.
```suggestion
* `sub` (the principal that is the subject of the JWT, [RFC7519, Section 4.1.2](https://tools.ietf.org/html/rfc7519#section-4.1.2)) — there are several possible formats:
  * If the **Client Account** is a muxed account (`M...`), the `sub` value should be the muxed account (`M...`).
  * If the **Client Account** is a stellar account (`G...`):
     * And, a memo was attached to the challenge transaction, the `sub` should be the stellar account appended with the memo, separated by a colon (`G...:17509749319012223907`).
     * Otherwise, the `sub` value should be Stellar account (`G...`).
```